index.html

```diff
@@ -18,21 +18,15 @@
 <body>
         <header>
             <a href="/" class="logo">My Logo</a>
-            <!-- <nav class="menu" id="icon"><i class="fa fa-bars "></i> -->
-                <nav class="navbar">
+            <nav class="menu" id="icon"><i class="fa fa-bars "></i></div>
+                
```

Comment:
Good job working on the mobile menu 👍. however, your mobile menu doesn't work,

- [ ] Kindly make sure you created the `.js` file then implement the following features : 
   - When the user clicks (or taps) the hamburger button, the mobile menu appears.
   - When the user clicks (or taps) the close (X) button, the mobile menu disappears.
   - When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
   - When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.